docs/_reference/use-cases/ai-app.md

```diff
@@ -0,0 +1,72 @@
+  * If you need more time for live Zap executions, you can request an extension by contacting Zapier Partner Support through their [contact form](https://zapier.com/partner/contact/). However, we can only do this so much and the most foolproof way would be to implement the previous recommendation.
+  * **Why:** Extending the execution time for live Zaps can help accommodate actions that inherently take longer to process.
+
+### 2. Handling Samples for Long-Running Tasks  
+
+**Pain Point:** Generating accurate samples for long-running tasks can be tricky, especially during the initial setup.
+
+**Recommendation:**
+
+* Use `bundle.meta.isLoadingSample`:  
+  * When `bundle.meta.isLoadingSample` is true, return a simplified or cached version of the data that represents a typical response.  
+  * **Why:** This approach ensures that users get a quick response during setup, avoiding delays caused by the actual long-running task.
+
+### 3. Hiding Complex Fields in Actions  
+
+**Pain Point:** Complex configuration fields can overwhelm users, making the setup process cumbersome.
+
+**Recommendation:**
+
+* Use Custom Input Fields:  
+  * You can create an input field that depends on another. For example, you could create an “Advanced Features” input field at the bottom of all of your other ones that has the property `altersDynamicFields: true` so when it’s updated, all of the other fields refresh. Then for any some additional input fields, they would be dependent if that “Advanced Features” input field is true, then display the more advanced ones that are not necessarily required. (ie. `if (bundle.inputData.advanced === true)`)
+  * These custom input fields can hide complex fields behind simpler user interfaces.  
+  * **Why:** Simplifying the user interface makes it more user-friendly, leading to a better experience and fewer setup errors.
+
+### 4. Use-Case Specific Actions  
+
+**Pain Point:** Users may struggle with setting up generalized actions that require specific configurations.
+
+**Recommendations:**
+
+* Create Use-Case Specific Actions:  
+  * For example, a “ChatGPT Summarize Text" action can hide the configuration and prompt details behind the scenes. This can be accomplished by hardcoding a prompt into the action configuration itself and then just have the user input the text that needs to be summarized.
+  * Why: Pre-configured actions tailored to specific use cases streamline the user experience, making it easier for users to set up and use your app effectively.
+* Create Use-Case Specific Dropdown:  
+  * For example, a “Send Prompt" action could have a dropdown of available “use-cases” that when selected, pre-fill input fields for the user. This can be accomplished by having a mapping of sorts for your use-cases and some complex logic using `altersDynamicFields: true` to then have all of your input fields as custom ones which are loaded in via functions based on the the use-case selected and the default of them pre-filled.
```

Comment:
include a link to https://github.com/zapier/zapier-platform/blob/main/packages/schema/docs/build/schema.md#fieldschema